go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/hassanjawwad12/betterj
+
+go 1.23.4
```

Comment:
Put an extra empty line at end and do it in all other files as well.

Reply:
The issue I am facing with this is that, when the file is saved , the code is automatically formatted till the last line and the extra line space is removed